bears/r/FormatRBear.py

```diff
@@ -14,8 +19,51 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         r_keep_comments: bool=True,
+                         r_keep_blank_lines: bool=True,
+                         r_braces_on_next_line: bool=False,
+                         r_use_arrows: bool=False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH):
```

Comment:
ah no don't remove the `r_max_expression_length` option, just make a default value that disables the passing to `tidy_source`! maybe `0` could be the default value that you check for below and then put it inside the rcode if wanted. I would suggest now to put this into a new separate commit, or if you even want into a new PR :+1: